run/src/adapt.ts

```diff
@@ -1,15 +1,24 @@
 import {Stream} from 'xstream';
 
+declare var window: any;
+
+function getGlobal(): any {
+  if(typeof window !== undefined) {
+    return window;
+  }
+  return global;
+}
```

Comment:
`typeof window !== undefined` will always be true because it should be `typeof window !== 'undefined'` instead. Also we should do the same typeof check for `global`, returning `this` as the last option.